### PR TITLE
Admin can edit album #54

### LINKED ISSUE #54: Admin edits an item
Background: an existing item
      As an admin
      When I visit "admin/items"
      And I click "Edit"
      Then my current path should be "/admin/items/:ITEM_ID/edit"
      And I should be able to upate title, description, image, and status

### PR DESCRIPTION
**Pull request by scottfirestone**
I created the ability for admins to edit albums, with the necessary routes, view, and controller action. Because it makes sense to redirect to the admin albums page once the album is successfully edited, I revised the new action to have the same behavior.

closes #54 

@Draganovic @Salvi6God @julsfelic

### CODE REVIEW

**Comment by scottfirestone** [comment]
I also deleted another feature test for admin not being able to create an invalid album as we should probably just handle this at the model level... right @julsfelic?

**Comment by julsfelic** [comment]
What you would want to do is have at least one feature test that would test the sad path. Specifically, you would test that just one of the validations (i.e. No album title) would fail. Then you would test the rest through model tests. @scottfirestone

File: app/views/admin/albums/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<section class="container section-container order-section">
+  <h1>Album Info</h1>
```
Comment (julsfelic) [inline app/views/admin/albums/index.html.erb]: Would this be plural since it is the index?

**Comment by julsfelic** [comment]
Overall looks really good. Left a small comment but besides that it is ready to be merged in.

**Comment by scottfirestone** [comment]
Thanks @julsfelic. I'm going to fix that per your comment and add a card for testing the sad path, and merge it.

**Pull request merged**